Controller/Component/WizardComponent.php

```diff
@@ -510,6 +529,9 @@ protected function _validStep($step) {
  * @return void
  */
 	protected function _setCurrentStep($step) {
+		if (!in_array($step, $this->steps)) {
+			return;
+		}
```

Comment:
I'm not a fan of checks like this. I mean I'd be concerned where they got the step from if it's an invalid choice, this is sort of an error you should never reach.